legal-api/src/legal_api/reports/report.py

```diff
@@ -708,10 +713,14 @@ def _get_party_name(party_json):
         party_name = ''
         if party_json.get('officer').get('partyType') == 'person':
             last_name = party_json['officer'].get('lastName')
-            first_name = party_json['officer'].get('firstName')
-            middle_initial = party_json['officer'].get('middleInitial')\
-                if party_json['officer'].get('middleInitial') else ''
-            party_name = f'{last_name}, {first_name} {middle_initial}'
+            first_name = party_json['officer'].get('firstName')\
+                if party_json['officer'].get('firstName') else ''
+            middle_name = party_json['officer'].get('middleName')\
+                if party_json['officer'].get('middleName') else ''
+            if not middle_name and not first_name:
+                party_name = f'{last_name}'
+            else:
+                party_name = f'{last_name}, {first_name} {middle_name}'
```

Comment:
There are some scenarios where we might have a value for `middleInitial`.  We should look at `middleName` and `middleInitial` just to be safe.

Reply:
so in this case I should check whether we have value for `middleInitial` or `middleName` , and use either of them

Reply:
yes